build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -86,11 +118,6 @@ data:
     # Service traffic.
     #  AntreaProxy: true
 
-    # Enable EndpointSlice support in AntreaProxy. Don't enable this feature unless that EndpointSlice
```

Comment:
Why remove endpointslice here?

Reply:
This has been removed according to https://github.com/antrea-io/antrea/pull/5393/files#diff-820e743fbc6549f822a09b2968d46254f0376b3889de673ba58fd4e85da0c8c1
I have rebased my branch

Reply:
Got it, thanks.